src/Essentials/src/Flashlight/Flashlight.ios.cs

```diff
@@ -6,6 +6,17 @@ namespace Microsoft.Maui.Devices
 {
 	class FlashlightImplementation : IFlashlight
 	{
+		/// <summary>
+		/// Checks if the flashlight is available and can be turned on or off.
+		/// </summary>
+		/// <returns><see langword="true"/> when the flashlight is available, or <see langword="false"/> when not</returns>
+		public Task<bool> CheckIsSupported()
+		{
+			var captureDevice = AVCaptureDevice.GetDefaultDevice(AVMediaTypes.Video);
+			return captureDevice != null &&
```

Comment:
We are trying to move away from == null or != null
```suggestion
			return captureDevice is not null &&
```